bin/improver-percentile

```diff
@@ -49,7 +51,10 @@ def main():
         "(member) data into percentiled data, but may calculate over any "
         "dimension coordinate. Alternatively, calling this CLI with a cube "
         "containing probabilities will convert those to percentiles using "
-        "the ensemble copula coupling plugin.")
+        "the ensemble copula coupling plugin. If no particular percentiles "
+        "are given at which to calculate values or no number of percentiles "
```

Comment:
"nor a number of percentiles is given..."?

Reply:
Clarified.